corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_select2.scss

```diff
@@ -168,3 +168,7 @@
   border-color: $input-disabled-bg !important;
   background-color: $input-disabled-bg !important;
 }
+
+.select2-container--open .select2-dropdown {
```

Comment:
@biyeun Flagging this because z-index changes are historically buggy. See https://github.com/dimagi/commcare-hq/pull/34452/commits/f82b21ba6a52129ffed6c9c2e442766aa8adf741 for context.